sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -1701,15 +1720,27 @@ class MeanAbsolutePercentageError(BaseForecastingErrorMetricFunc):
         If array-like, values used as weights to average the errors.
         If 'raw_values', returns a full set of errors in case of multioutput input.
         If 'uniform_average', errors of all outputs are averaged with uniform weight.
+    multilevel : {'raw_values', 'uniform_average', 'uniform_average_time'}
+        Defines how to aggregate metric for hierarchical data (with levels).
+        If 'uniform_average' (default), errors are mean-averaged across levels.
+        If 'uniform_average_time', errors are mean-averaged across rows.
+        If 'raw_values', does not average errors across levels, hierarchy is retained.
 
     See Also
     --------
     MedianAbsolutePercentageError
     MeanSquaredPercentageError
     MedianSquaredPercentageError
 
+    Notes
+    -----
+    In some literatures, sMAPE is also known as Adjusted MAPE because
+    ,in some cases, over- and under-forecasts are not really penalised equally. [1]_
```

Comment:
do you have the exact page or section number?

Does this contain both the sMAPE and adj MAPE terminology?

Reply:
My intention is to provide readers with additional resources to understand why symmetric MAPE is not truly "symmetric." This explains the pushback against naming it as such. Perhaps I should include a section titled "Further Reading" instead of listing it under "References"?

Reply:
I think "references" is perfectly fine, if you list it as a reference for the statements in relation to the terminology "symmetric".

Having said this, the symmetric MAPE *is* symmetric, no? It satisfies the definition of "symmetric function" (see [wikipedia](https://en.wikipedia.org/wiki/Symmetric_function)) if you consider the prediction vector and training data vector as individual variables. Similarly, the individual sample losses are symmetric in the prediction and training data point.

Reply:
fair point. I suppose I'll remove the reference. rather provide a flag to readers something along the line '_one subtle problem to note is that sMAPE is not symmetric in the sense that over- and under-forecasts are not treated equally. i.e  over-forecasting with y_t = 150 and yhat_t = 200 gives sMAPE = 0.29, while under-forecasting with y_t = 150 and yhat_t = 100 gives sMAPE = 0.4. However, MAPE = 0.33 for both cases_.'. Also, based on this [Wikipedia](https://en.wikipedia.org/wiki/Symmetric_mean_absolute_percentage_error), it appears adjusted MAPE equation is slightly different than the symmetric MAPE equation we have in the library, namely on the denominator.

Reply:
thanks - this is making more sense then.

What I would avoid though is too extensive discussion of the metrics, which one is better or problematic etc, such as what follows "one subtle problem" (the "asymmetry" is a general problem for relative metrics).

This is better done in scientific papers (which can be referred to).
You could add the original references though, and/or references of papers that discuss it.

Reply:
In light of the wikipedia page, I've removed a discussion on adjusted MAPE. Kindly check the latest changes :)